### PR TITLE
made both type and page.name lowercase when looking up URLs

### PR DESCRIPTION
**Pull request by adamgibs**

### CODE REVIEW

File: apps/main-app/src/components/dataTable/getResourceUrl.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ export const getResourceUrl = (curie, type) => {
   } else {
     [resource] = resourceDescriptors
         .filter(resource => resource.db_prefix === prefix)[0].pages
-        .filter(page => page.name === type.toLowerCase())[0].url
+        .filter(page => page.name.toLowerCase() === type.toLowerCase())[0].url
```
Comment (oblodgett) [inline apps/main-app/src/components/dataTable/getResourceUrl.js:23]: page.name is already lowercase? What does this fix?

Comment (adamgibs) [reply apps/main-app/src/components/dataTable/getResourceUrl.js:23]: No all of them. The one that was failing for zfin was "Fish"

Comment (oblodgett) [reply apps/main-app/src/components/dataTable/getResourceUrl.js:23]: Ahh ok... this is wrong in the resource Descriptor file:
https://github.com/alliance-genome/agr_schemas/blob/master/resourceDescriptors.yaml#L183

Comment (adamgibs) [reply apps/main-app/src/components/dataTable/getResourceUrl.js:23]: So should this still go in? It want hurt anything if this is later fixed in the agr_schemas

**Pull request closed without merging**